### PR TITLE
Feature/check details acceptance tests

### PR DESCRIPTION
**Pull request by mikeSmith08**

### CODE REVIEW

File: src/test/acceptance/features/check-details.feature [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Feature: Complete application journey
+  In order to apply for the HTBHF programme
+  As a potential claimant
+  I want to enter my details and complete the application journey
+
+  Scenario: Valid application details for pregnant woman are shown on the check details page
```
Comment (dwybourn) [inline src/test/acceptance/features/check-details.feature]: a pregnant woman

File: src/test/common/page/data-entry-page.js [hunk 2]
```diff
@@ -9,6 +9,11 @@ class DataEntryPage extends Page {
     const submitButton = await this.getSubmitButton()
     await submitButton.click()
   }
+
+  async getSubmitButtonText () {
```
Comment (dwybourn) [inline src/test/common/page/data-entry-page.js]: We should change this class name.

Comment (mikeSmith08) [reply src/test/common/page/data-entry-page.js]: Done

File: src/test/acceptance/features/check-details.feature (hunk 1 above)
Comment (mikeSmith08) [reply src/test/acceptance/features/check-details.feature]: Done

File: src/test/acceptance/features/check-details.feature [hunk 3]
```diff
@@ -0,0 +1,23 @@
+Feature: Complete application journey
```
Comment (YetAnotherMatt) [inline src/test/acceptance/features/check-details.feature]: The feature description needs to be updated for check details

File: src/test/common/page/are-you-pregnant.js [hunk 4]
```diff
@@ -1,7 +1,7 @@
 'use strict'
 const webdriver = require('selenium-webdriver')
 
-const DataEntryPage = require('./data-entry-page')
+const SubmitablePage = require('./submitable-page')
```
Comment (YetAnotherMatt) [inline src/test/common/page/are-you-pregnant.js]: Spelling - should be 'submittable' (two t's)

**Pull request merged**